src/CategoryCard/CategoryCard.scss

```diff
@@ -1,7 +1,15 @@
 .category-card {
+  background-color: black;
+  color: white;
   height: 36%;
-  width: 19%;
-  border: 5px solid black;
+  width: 30%;
+  padding: 30px;
+  margin: 25px;
+  border-radius: 5px;
+  text-transform: uppercase;
+  font-weight: 900;
+  box-shadow: 5px 5px 0px rgba(0, 0, 0, .25);
```

Comment:
Thank you for teaching us about hex codes and how the numbers are decided and what they mean!